index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<!-- <link rel="stylesheet" href="./dist/index.css" /> -->
+		<link rel="stylesheet" href="/scss/dist/index.css" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+			rel="stylesheet"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+		/>
+		<title>Document</title>
+	</head>
+	<body>
+		<header id="header">
+			<div class="top-banner">
+				<div class="banner">
+					<img src="./images/Facebook.svg" alt="" />
+					<img src="./images/twitter.svg" alt="" />
+					<p>Logout</p>
+					<p>My Page</p>
+					<p>English</p>
+				</div>
+			</div>
+			<div class="menu" id="mobile-menu">
+				<div class="menu-bottom">
+					<img src="./images/logo.svg" alt="" />
+					<div class="left-menu">
+						<ul class="left">
+							<li><a class="active" href="#">Home</a></li>
+							<li><a href="./about.html">About</a></li>
+							<li><a href="">Program</a></li>
+							<li><a href="">News</a></li>
+						</ul>
+						<button>CC compaign</button>
+					</div>
+				</div>
+			</div>
+			<div id="humburger">
+				<span class="material-symbols-outlined"> menu </span>
+			</div>
+		</header>
+		<section id="headline">
+			<div class="content">
+				<div class="heading">
+					<h3>“Hello! Sharing world”</h3>
+					<h1>2021 Glocal Mission Summit</h1>
+				</div>
+				<div class="discription">
+					<p>
+						The Glocal Mission Summit is a biannual event where
+						people engage in conversations around mission and
+						imagine innovative ways to engage and embrace our local
+						communities that are becoming increasingly diverse.
+					</p>
+				</div>
+				<div class="location">
+					<h4>2021.10.15(THU) ~ 16(FRI)</h4>
+					<p>@ National Museum of Korea, Art Center Nabi and more</p>
+				</div>
+			</div>
+		</section>
+		<section id="mainProgram">
+			<div class="program">
+				<div class="title">
+					<h4>Main Program</h4>
+					<hr />
+				</div>
```

Comment:
> You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
- Colors.

- [ ] Great job implementing this section. As stated in the project requirement, you need to follow all the design guidelines including the color, The lines below these titles don't match the color in the Given design, so I'll suggest you update these. :small_airplane: 

- - Your design:
![Screenshot from 2023-06-29 14-21-21](https://github.com/sheezu900/module1-capstone-project/assets/107632780/b290f408-c29d-4727-b4ac-35a6c71f792e)
![Screenshot from 2023-06-29 14-21-32](https://github.com/sheezu900/module1-capstone-project/assets/107632780/b4f5043e-11ca-4ec0-af9b-730cc1d07a10)

- - GIVEN DESIGN:
![Screenshot from 2023-06-29 14-22-06](https://github.com/sheezu900/module1-capstone-project/assets/107632780/effcc2d9-be31-4241-b379-6cae331bd130)